airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -186,7 +188,13 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
   private NormalizationInput generateNormalizationInput(final StandardSyncInput syncInput,
                                                         final StandardSyncOutput syncOutput) {
 
-    return normalizationActivity.generateNormalizationInput(syncInput, syncOutput);
+    final int version = Workflow.getVersion(REMOVE_ACTIVITY_FOR_NORM_INPUT, Workflow.DEFAULT_VERSION, REMOVE_ACTIVITY_FOR_NORM_INPUT_CURRENT_VERSION);
+    if (version == Workflow.DEFAULT_VERSION) {
+      return normalizationActivity.generateNormalizationInput(syncInput, syncOutput);
+    } else {
+      return normalizationActivity.generateNormalizationInputWithMinimumPayload(syncInput.getDestinationConfiguration(), syncInput.getCatalog(),
```

Comment:
instead of 'syncInput.getCatalog()', should this instead be `syncOutput.getOutputCatalog()`? The existing `normalizationActivity.generateNormalizationInput` implementation uses the output catalog instead of the input catalog, so I wanted to call this out